lib/simple-seeder.js

```diff
@@ -5,6 +5,9 @@ const speedometer = require('speedometer')
 const Id = require('hypercore-id-encoding')
 const mutexify = require('mutexify/promise')
 const SeedBee = require('seedbee')
+const { contentEncoding } = require('seedbee/encoding')
+
+const allowedPeersKey = 'simple-seeder/allowed-peers'
```

Comment:
Can this be `ALLOWED_PEERS` in mayus? So when reading the code below we know right away it's a real const

Reply:
Also ok to write the string twice but any way is ok